src/framework/actions/turn.ts

```diff
@@ -3,6 +3,18 @@ import { discardAll, drawCard } from "./hand";
 import { MAX_GOLD, MAX_HAND_SIZE, WORKERS_TO_SKIP_TECH } from "../constants";
 import { queryInstances } from "../queries/common";
 
+export const startGame = ($: GameState): void => {
```

Comment:
nit: We know that the first call to `handleReadyPhase` will never "block" on user input, and so can be executed with no problems. Is there any reason to force the caller to always make two calls? Why not just have `createNewGameState` return a state that is already in the first player's main phase (perhaps by calling `handleReadyPhase` itself)?

I think it's also fine to leave this as-is, but it does seem like a slightly strange choice. Maybe it comes in handy for animating turn 1 income in the UI or sth (but that's questionable, since the UI will need special handling for turn 1 regardless)

Reply:
I was thinking of it from both an animation perspective and a "Click to start your turn/start the game" kind of perspective. I don't have a strong opinion either way though.

Reply:
I left this as it is for now.